package.json

```diff
@@ -17,7 +17,7 @@
     "lint": "eslint . && prettier --check '**/*.{js,md}'",
     "test": "npm run test-node && npm run test-docs",
     "test-browser": "npm run build && npm run bundle && npm run build-tests && npm run bundle-tests && karma start --single-run --browsers ${BROWSER:-ChromeHeadless}",
-    "test-docs": "mocha `find documentation -name '*.md'`",
+    "test-docs": "mocha --require unexpected-markdown `find documentation -name '*.md'`",
```

Comment:
Shouldn't this use `evaldown --validate`?

Reply:
Not yet - I’ve not gotten per snippet output implemented yet which means swapping would be a regression. But should be a breeze to add later :)

Reply:
Ah, okay :+1:

Reply:
...to clarify, what I mean is at the level of an individual output line via the spec reporter. But the stringified error report is obviously present, so if we can live with that we could switch now and it’ll get better at some future release.

Reply:
I'm happy to make the switch now given that the feature is coming :)